### PR TITLE
docs: update workflow bot readme

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

File: templates/bot/js/workflow/README.md [hunk 1]
```diff
@@ -1,103 +1,74 @@
...
 > Your app can be installed into a team, or a group chat, or as personal app. See [Installation and Uninstallation](https://aka.ms/teamsfx-command-response#customize-installation).
 >
 
-# Understanding the code
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+2. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+3. Press F5 to start debugging which launches your app in Teams using a web browser. Select `Debug (Edge)` or `Debug (Chrome)`.
+4. When Teams launches in the browser, select the Add button in the dialog to install your app to Teams.
+5. Type or select `helloWorld` in the chat to send it to your bot - this is the default command provided by the template.
+6. In the response from the bot, select the **Do Stuff** button.
```
Comment (dooriya) [inline templates/bot/js/workflow/README.md:27]: **Do Stuff** -> **DoStuff** to keep consistent with the `title` defined in adaptive card.

File: templates/bot/js/workflow/README.md [hunk 2]
```diff
@@ -165,9 +134,19 @@ export class Handler1 {
...
+> Please note:
+>
+> - The `triggerVerb` is the `verb` property of your action.
+>
+> - The `actionData` is the data associated with the action, which may include dynamic user input or some contextual data provided in the `data` property of your action.
+>
+> - If an Adaptive Card is returned, then the existing card will be replaced with it by default.
 
-### Step 4: register the action handler
+You can customize what the action does here, including calling an API, processing data, etc.
```
Comment (dooriya) [inline templates/bot/js/workflow/README.md:145]: missing link for `here`?

File: templates/bot/js/workflow/README.md [hunk 3]
```diff
@@ -165,9 +134,19 @@ export class Handler1 {
 }
```
Comment (dooriya) [inline templates/bot/js/workflow/README.md:134]: Need to update the JavaScript code snippet here, you can check @a1exwang's pr here: https://github.com/OfficeDev/TeamsFx/pull/6326

File: templates/bot/js/workflow/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply templates/bot/js/workflow/README.md]: done

File: templates/bot/js/workflow/README.md (hunk 3 above)
Comment (MuyangAmigo) [reply templates/bot/js/workflow/README.md:135]: Done

**Review by dooriya — APPROVED** [review]

**Review by yukun-dong — APPROVED** [review]

**Pull request merged**